lib/aur-sync

```diff
@@ -144,6 +144,9 @@ while true; do
             makepkg_args+=(--log) ;;
         -P|--provides)
             provides=1 ;;
+        --provides-from)
+            shift; IFS=, read -a repo -r <<< "$1"
+            repo_p+=("${repo[@]}") ;;
```

Comment:
Isn't `read -a repo_p -r <<< "$1"` enough here?

Reply:
I guess it's too late now, but I copied it from `--ignore`, where the idea was to combine all the arguments into one. So `--provides-from=a,b --provides-form=c,d` would be the same as `--provides=a,b,c,d`. With `read -a`, it'd be whatever `--provides-from` is specified last.

Reply:
Makes sense. Wasn't thinking in the multiple `--provides-from` case. :+1: